integration-tests/src/test_loop/utils/transactions.rs

```diff
@@ -56,12 +57,15 @@ pub(crate) fn get_anchor_hash(clients: &[&Client]) -> CryptoHash {
 }
 
 /// Get next available nonce for the account's public key.
-pub fn get_next_nonce(env: &mut TestLoopEnv, account_id: &AccountId) -> u64 {
+pub fn get_next_nonce(
+    test_loop_data: &TestLoopData,
+    node_datas: &[TestData],
```

Comment:
Just out of curiosity why this change? Did you have some trouble with mut?

Reply:
It's because I didn't have `TestLoopEnv` for event handling, and I also removed  `mut` because it is read-only call.